src/middlewared/middlewared/sqlalchemy.py

```diff
@@ -0,0 +1,150 @@
+import datetime
+import json
+
+from sqlalchemy import (
+    Table, Column as _Column, ForeignKey, Index,
+    Boolean, CHAR, DateTime, Integer, SmallInteger, String, Text,
+)  # noqa
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship  # noqa
+from sqlalchemy.types import UserDefinedType
+
+from middlewared.plugins.pwenc import encrypt, decrypt
+
+
+def django_ix(index, table):
```

Comment:
Can you give a quick explanation for dummies why that is needed to be kept in code and not migrated once?

Reply:
If we don't do this, a big useless migration will appear that will re-create indexes. We have to choose one of the evils and I guess I've chosen the wrong one?

Reply:
Well, from my point of view it seems to be more sane to have one big useless migration to keep the original code cleaner without any mentions of "django" : P
Thats just how I see it though, I wont mind too much if you dont think its worth it.